mobile-app/cypress/integration/functional/wallet/balances/announcements.spec.ts

```diff
@@ -116,4 +117,145 @@ context('Wallet - Balances - Announcements', () => {
     cy.getByTestID('announcements_banner').should('exist')
     cy.getByTestID('announcements_text').should('contain', 'Guidelines')
   })
+
+  it('should display blockchain is down warning message with no existing announcement', () => {
```

Comment:
Shall we add a `context()` or `describe()` to group blockchain down-related announcement tests?